src/modules/dashboard/components/SubscribedColoniesList/SubscribedColoniesDropdown.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import ColonyAvatar from '~core/ColonyAvatar';
+import DropdownMenu, {
+  DropdownMenuSection,
+  DropdownMenuItem,
+} from '~core/DropdownMenu';
+import Popover from '~core/Popover';
+import { ProcessedColony } from '~data/generated';
+import styles from './SubscribedColoniesList.css';
+
+const displayName =
+  'dashboard.SubscribedColoniesList.SubscribedColoniesDropdown';
+
+export type Colony = Pick<
+  ProcessedColony,
+  | 'colonyName'
+  | 'colonyAddress'
+  | 'id'
+  | 'displayName'
+  | 'avatarHash'
+  | 'avatarURL'
+>;
+interface Props {
+  activeColony: Colony;
+  coloniesList: Colony[];
+}
+
+const SubscribedColoniesDropdown = ({ activeColony, coloniesList }: Props) => {
+  return (
+    <Popover
+      content={() => {
+        return (
+          <DropdownMenu>
+            <DropdownMenuSection>
+              {coloniesList.map((colony) => (
+                <DropdownMenuItem key={colony.colonyAddress}>
+                  <NavLink
+                    activeClassName={styles.activeColony}
+                    title={colony.colonyName}
+                    to={`/colony/${colony.colonyName}`}
+                  >
+                    <div className={styles.dropdownItem}>
+                      <div className={styles.itemImage}>
+                        <ColonyAvatar
+                          colony={colony}
+                          colonyAddress={colony.colonyAddress}
+                          size="xs"
+                        />
+                      </div>
```

Comment:
![Screenshot from 2022-06-23 22-30-15](https://user-images.githubusercontent.com/1193222/175382732-7119fb83-59aa-4c53-8ee7-542e5c75a5dd.png)

I think we need less padding between the elements. This is more of a think for Product, rather than your implementation